### ISSUE TITLE
Edit with s2edit?

### ISSUE THREAD

**Issue by carpusmedia** [Merri/map-generator#4]
Is it possible to edit the maps generated with s2edit?

**Comment by Merri** [comment]
Yes, you can edit the maps with s2edit and it is actually highly recommended too since the generator doesn't add any decorative stuff on the map :)
